app/models/product.rb

```diff
@@ -0,0 +1,29 @@
+# == Schema Information
+#
+# Table name: products
+#
+#  id              :bigint           not null, primary key
+#  description     :string
+#  name            :string           not null
+#  jira_project_id :bigint
+#
+# Indexes
+#
+#  index_products_on_jira_project_id  (jira_project_id)
+#  index_products_on_name             (name)
+#
+# Foreign Keys
+#
+#  fk_rails_...  (jira_project_id => jira_projects.id)
+#
+
+class Product < ApplicationRecord
+  belongs_to :jira_project
```

Comment:
hmm so the product BELONGS to a JIRA project? Doesn't sound correct to me. Instead, I think the `Product` to be the root of all models, and that the JIRA project BELONGS to a product.. thoughts?

Reply:
I didn't think of model hierarchy, just created the relationship.  I'll change it so we could have a product even if the jira project haven't been created yet.